### ISSUE TITLE
Proximity button abuse

### ISSUE THREAD

**Issue by BrotherNova** [Eggs-D-Studios/wos-issues#548]
### Guidelines

- [X] I checked for duplicate bug reports
- [X] I tried to find a way to reproduce the bug

### Version

Main (Production)

### What happened? What did you expect to happen?

A stack of proximity buttons set to the max activation range and lowest activation time, with about 6 or 7 buttons corresponding to "asdwe[Space]" and esc can stunlock a player

### How can we reproduce this bug?

Make a stack of proximity buttons at max range with all keys that could be used to escape, selected.

### Frequency

Always

**Comment by dogranger1** [comment]
https://github.com/user-attachments/assets/f8d13ef8-4455-44b4-82ba-cee3ff687807

**Comment by BrotherNova** [comment]
They can all be stacked and then duplicated 8 times, put these into 8 corners you can tilt your head, since these can also be set for long distances... Stunlocking entire ships with just 8 little drones is easy. Too easy.

**Comment by Pasha82831** [comment]
(To developers) Best way to fix it: set config limit

**Comment by hail12pink** [comment]
> (To developers) Best way to fix it: set config limit

there is already a limit to the range

**Comment by BrotherNova** [comment]
Yes but its very large, more than what I would think would be needed for any application of this.

**Comment by hail12pink** [comment]
> Yes but its very large, more than what I would think would be needed for any application of this.

lowering range limit doesn't fix anything here

**Comment by hail12pink** [comment]
https://github.com/Eggs-D-Studios/Waste-of-Space/commit/c105b9b7f926180b50ba01d49c3109849e0253b7

**Comment by LuaBloxor** [comment]
Also this error due to unsupported Keybind support is unhandled

<img width="1263" height="74" alt="Image" src="https://github.com/user-attachments/assets/ec9fb3a2-a130-4058-bdf8-ed896faa58c7" />
